ansible_collections/arista/avd/roles/eos_designs/docs/wan-preview.md

```diff
@@ -75,7 +76,28 @@ The intention is to support both a single [AutoVPN design](https://www.arista.co
 
 ###### HA
 
-To Be Implemented.
+for eBGP LAN routing protocol the following is done to enable HA:
+
+- the HA interface(s) is(are) the uplink interface(s).
+- the HA interface(s) subnet(s) are redistributed in BGP via the `RM-CONN-2-BGP` route-map
+- BGP underlay peer group is configured with `allowas-in 1` to be able to learn the HA peer uplink interface subnet coming with the same ASN via eBGP over the LAN.
```

Comment:
```suggestion
BGP underlay peer group is configured with `allowas-in 1` to be able to learn the HA peer uplink interface subnet over the LAN as well as learning WAN routes from other sites (as backup in case all WAN links are lost).
```